### PR TITLE
Feat(frontend):  shorten date notification wall

### PR DESCRIPTION
**Pull request by MariePrandstetter**
#4850 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/src/container/NotificationWall.jsx [hunk 1]
```diff
@@ -40,6 +40,16 @@ import {
...
 import { escape as escapeHtml } from 'lodash'
 
+function shortDate (date) {
+  const secondsElapsed = Date.now() - new Date(date).getTime()
+  if (secondsElapsed < 60000) return Math.round(secondsElapsed / 1000) + ' sec'
+  if (secondsElapsed < 3600000) return Math.round(secondsElapsed / 60000) + ' min'
+  if (secondsElapsed < 3600000 * 24) return Math.round(secondsElapsed / 3600000) + ' h'
+  if (secondsElapsed < 3600000 * 24 * 30) return Math.round(secondsElapsed / (3600000 * 24)) + ' j'
+  if (secondsElapsed < 3600000 * 24 * 365) return Math.round(secondsElapsed / (3600000 * 24 * 30)) + ' m'
+  return Math.round(secondsElapsed / (3600000 * 24 * 365)) + ' a'
```
Comment (gbonaspetti) [inline frontend/src/container/NotificationWall.jsx:50]: These strings must be translated.

File: frontend/src/container/NotificationWall.jsx [hunk 2]
```diff
@@ -467,18 +477,25 @@ export class NotificationWall extends React.Component {
...
-                            escapeHtml(displayDistanceDate(notification.created, props.user.lang)) +
                           '</span>'
                         )
                       }}
                     />
                   </div>
-                  {!notification.read && <i className='notification__list__item__circle fas fa-circle' />}
+                  <div className='notification__list__item__meta'>
+                    <span className='notification__list__item__meta__date'
+                          title={formatAbsoluteDate(notification.created, props.user.lang)}>
```
Comment (gbonaspetti) [inline frontend/src/container/NotificationWall.jsx:492]:
```suggestion
                    <span 
                      className='notification__list__item__meta__date'
                      title={formatAbsoluteDate(notification.created, props.user.lang)}
                     >
```

File: frontend/src/css/NotificationWall.styl [hunk 3]
```diff
@@ -37,16 +37,27 @@
         margin standardSpacing
         display inline-block
         width 30px
-      &__text
+       &__text
```
Comment (gbonaspetti) [inline frontend/src/css/NotificationWall.styl:40]: Is the tabulation right? It seems that there is one space added.

File: frontend/src/css/NotificationWall.styl [hunk 4]
```diff
@@ -37,16 +37,27 @@
...
+        flex 1
         display flex
-        padding-left 5px
-        padding-right standardSpacing
+        font-size metadataFontSize
+        word-break break-word
+        align-items center
+      &__meta
+        padding-right 5px
```
Comment (gbonaspetti) [inline frontend/src/css/NotificationWall.styl:47]:
```suggestion
        padding-right textSpacing
```
?

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/src/css/NotificationWall.styl (hunk 3 above)
Comment (MariePrandstetter) [reply frontend/src/css/NotificationWall.styl:40]: It was a mistake, thanks

**Review by gbonaspetti — APPROVED** [review]

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**